app/src/main/java/com/beemdevelopment/aegis/ui/fragments/preferences/AuditLogPreferencesFragment.java

```diff
@@ -64,10 +62,10 @@ public void onViewCreated(@NonNull View view, Bundle savedInstanceState) {
             _noAuditLogsView.setVisibility(entries1.isEmpty() ? View.VISIBLE : View.GONE);
 
             for (AuditLogEntry entry : entries1) {
-
                 VaultEntry referencedEntry = null;
-                if (entry.getReference() != null) {
-                     referencedEntry = _vaultManager.getVault().getEntryByUUID(UUID.fromString(entry.getReference()));
+                UUID referencedEntryUUID = UUID.fromString(entry.getReference());
```

Comment:
This line currently throws an error in case `entry.getReference()` returns null. This happens with events that don't require an entry reference (ie. vault unlock).